packages/datadog-plugin-http/src/server.js

```diff
@@ -59,6 +60,16 @@ class HttpServerPlugin extends Plugin {
 
       web.wrapRes(context, context.req, context.res, context.res.end)()
     })
+
+    this.addSub('datadog:tracer:trace:finish', ({ traces }) => {
+      if (!this.config.blocklist || traces.length === 0) return undefined
```

Comment:
`undefined` is redundant as it's the default.

Reply:
There should also be an additional check that the root span is actually an HTTP request.